docs/advanced_topics/api/index.md

```diff
@@ -1,17 +1,19 @@
-===========
-Wagtail API
-===========
+# Wagtail API
 
 The API module provides a public-facing, JSON-formatted API to allow retrieving
 content as raw field data. This is useful for cases like serving content to
 non-web clients (such as a mobile phone app) or pulling content out of Wagtail
 for use in another site.
 
-See `RFC 8: Wagtail API <https://github.com/wagtail/rfcs/blob/main/text/008-wagtail-api.md#12---stable-and-unstable-versions>`_
+See [RFC 8: Wagtail API][1]
 for full details on our stabilisation policy.
 
-.. toctree::
```

Comment:
the markup for this is a bit different, see https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#tables-of-contents

    ```{toctree}
    ---
    maxdepth: 2
    titlesonly:
    ---
    v2/configuration
    v2/usage
    ```

Reply:
I changed it to the syntax you suggested, but it's not displaying correctly in Github.
Is there something maybe I'm forgetting?

Reply:
I changed this particular section to match the suggestion